third_party/rust/crates/remote/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+# Export file for Stardoc support
```

Comment:
Should this file have the autogen tag? Is this something that was done by hand to enable Stardoc support?

Reply:
This probably _should_ have a tag, but this is exactly what was emitted by `cargo raze`.